client/vscode/src/activation.ts

```diff
@@ -78,7 +78,9 @@ export async function activateDocumentsLanguageServer(extensionContext: Extensio
         debugOptions.execArgv.push('--pre-init-encryption')
         // Used by the aws-language-server-runtime based servers
         debugOptions.execArgv.push('--set-credentials-encryption-key')
-        const child = cp.spawn('node', [serverModule, ...debugOptions.execArgv])
+
+        // --inspect or --inspect-brk has to be passed right after the node command, else debuggers won't be able to be attached to the process
```

Comment:
Is it because `inspect` was treated as a script argument instead of `node` argument? Then comment is a bit misleading.

Reply:
you're right, I'll add that bit as well

Reply:
Good question, I'm not entirely aware on what they do. I'll check first but these most probably will need to be moved as well